internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/TestPushJobWithNativeReplication.java

```diff
@@ -458,7 +460,9 @@ public void testNativeReplicationForHeartbeatSystemStores(int recordCount, int p
             TestWriteUtils.updateStore(
                 VPJ_HEARTBEAT_STORE_NAME,
                 parentControllerClient,
-                new UpdateStoreQueryParams().setLeaderFollowerModel(true).setNativeReplicationEnabled(true));
+                new UpdateStoreQueryParams().setActiveActiveReplicationEnabled(true));
```

Comment:
The AA config should be configured for this store by default instead of setting it up ad-hoc in tests. Can you check the setup routine on the controllers for this?